cmd/buildkitd/config/config.go

```diff
@@ -40,6 +40,7 @@ type GRPCConfig struct {
 type RegistryConfig struct {
 	Mirrors   []string `toml:"mirrors"`
 	PlainHTTP bool     `toml:"http"`
+	CA        string   `toml:"ca"`
```

Comment:
Name this `AdditionalCA` or `ExtraCA`.

Reply:
Shouldn't this be array?

Reply:
This is on a per registry basis, so I don't think supplying multiple CA certs would be usefull.

From my understanding a single registry can have only one cert, which can therefore be only signed by a single CA

Reply:
> This is on a per registry basis,

Correct. Ignore that comment